test/unit/vl-rich-data-table.test.html

```diff
@@ -119,6 +119,22 @@
             assert.equal(pager.totalItems, value.paging.totalItems);
         });
 
+        test('De paginatie van een tabel kan toch gezet worden via de data setter als er geen data is', () => {
```

Comment:
De navigatie is niet zichtbaar op dat moment, misschien kan dat nog toegevoegd worden aan de test als dit nog niet op een andere plaatst getest werd.